### ISSUE TITLE
[iOS] Can't pass by loading operating system nor resume image

### ISSUE THREAD

**Issue by luisgeraldes32** [nspire-emus/firebird#104]
I have an iPhone 5S, 9.3.3 jailbroken.
When I do start, it initializes the operating system, then it get's stuck here (with speed 0-5%):
![photo 17-10-17 19 01 39](https://user-images.githubusercontent.com/12972135/31681197-71d55ea4-b36e-11e7-8035-4e09dd827fee.png)

I can't neither resume my snapshot:
![photo 17-10-17 19 00 04](https://user-images.githubusercontent.com/12972135/31681208-783ca09a-b36e-11e7-818d-218e3aa7539f.png)

I got the boot's somewhere from the internet, got last official OS (4.5) from TI website and flashed on Windows (and it Works perfectly). Then i saved a snapshot on Windows and then passed all useful files to iPhone.

I have also an iPad Air 2 10.1.1 jailbroken.
In here, I can't resume my snapshot neither and, after pressing I to install OS and completes, the Nspire screen shuts down LCD and does nothing more.

Any suggestions?

Thanks in advance. 

EDIT: If I try to force close the app (in both devices), I can't open it anymore unless I uninstall, fully restart device (respring doesn't work), apply jailbreak, and then re-install. Because of need to apply jailbreak (especially on iPad) it's an hard and long time work!

**Comment by adriweb** [comment]
We will hopefully release a new version soon, it might fix it. I'll try to upload a test release here first.

In the meantime, I shall remind that due to a bug apparently in ARM64 Qt libraries, Firebird-Emu won't work on iOS11+.

**Comment by luisgeraldes32** [comment]
Thank you. I will test it as soon as you publish it here. I don't have iOS 11 so, no problem about that.

Anyway, I've tested it again on an android 7.0 and it turns off LCD after install OS.

**Comment by luisgeraldes32** [comment]
Re-tested on Android with that release found in another issue topic and it worked like a charm!
Only iOS remaining. Do you know +/- when upload the test release here?

Thanks

**Comment by adriweb** [comment]
Here's something to try (arm64 JIT version. iOS 10 minimum): [firebird-emu-iOS_1.4-test.ipa.zip (JIT)](https://github.com/nspire-emus/firebird/files/1435158/firebird-emu-iOS_1.4-test.ipa.zip)

I'll upload the noJIT version later.

Note: on my non-jailbroken iPhone 7 running iOS 11.1, the noJIT version works (but rather slowly, as expected), but the JIT version does not (crashes during initial JIT code running).
Note 2: because Qt appears to be broken with arm64, there are some weird workarounds in place, that allow things to run, but don't be surprised about odd characters/symbols in the GUI.

**Comment by luisgeraldes32** [comment]
Just tried it, as soon as I press start, the app crashes and I can't open it anymore, need to uninstall and reinstall.
iPad Air 2 10.1.1 jailbroken.

**Comment by adriweb** [comment]
Meh, I have a feeling we'll need to stick with Firebird 1.3 for iOS < 11, since the 32bit versions run fine there.
The 64bit version are probably too bug-riddled (partly because of Qt itself) and even worse on iOS 11 since the JIT refuses to work like on iOS 10 (from what I can see).

I might be able to build a 32bit version (maybe even JIT and noJIT) of FB 1.4 for iOS <= 10 but not tonight :/

**Comment by luisgeraldes32** [comment]
Well, when possible, post something so I can test it.
This 1.4version doesn't work on iOS 9, right?

**Comment by adriweb** [comment]
I could probably make it OK for iOS 9, but apparently the qt libs I'm using were made "for" iOS 10, so I'm not sure what would happen.

**Comment by luisgeraldes32** [comment]
So you need to change something in the ipa to work with iOS9 or I could probably try it?
It would be very useful to work on 9 because I'm using iOS 9 on iPhone, and I probably need it more than working on iPad with 10

**Comment by adriweb** [comment]
Okay, I was able to make some builds (and with iOS 9 compatibility in theory):

32bit JIT and noJIT: [firebird_iOS_32bit.zip](https://github.com/nspire-emus/firebird/files/1435764/firebird_iOS_32bit.zip)
64bit JIT and noJIT: [firebird_iOS_64bit.zip](https://github.com/nspire-emus/firebird/files/1435765/firebird_iOS_64bit.zip)

**Comment by adriweb** [comment]
OK we figured more things out and were able to get the JIT running, so expect more builds "soon™"

**Comment by luisgeraldes32** [comment]
Very nice, asap I'll try it on my iOS 9 iPhone 5S :)
Thank you

**Comment by adriweb** [comment]
Please try the [latest release](https://github.com/nspire-emus/firebird/releases/latest) and reopen the issue if needed (generic comment, might not be 100% related)
